kde-kit/autogen.kit.d/base.yml

```diff
@@ -0,0 +1,488 @@
+        >=kde-frameworks/extra-cmake-modules-5.115.0:*
+        virtual/pkgconfig
+      depend: |
+        dev-qt/qt5compat:6
+        dev-qt/qtbase:6[gui]
+        designer? ( dev-qt/qttools:6[designer] )
+        pulseaudio? (
+          dev-libs/glib:2
+          media-sound/pulseaudio[glib]
+        )
+      rdepend: |
+        ${DEPEND}
+        !media-libs/phonon-gstreamer
+        || (
+          kde-frameworks/breeze-icons:6
+          kde-frameworks/oxygen-icons:*
+        )
+      pdepend: |
+        vlc? ( >=media-libs/phonon-vlc-0.12.0 )
+
+  packages:
+    - phonon:
+        vars:
+          desc: KDE multimedia abstraction library
+          body: |
+            src_configure() {
+              use debug || append-cppflags -DQT_NO_DEBUG
+
+              local mycmakeargs=(
+                -DQT_MAJOR_VERSION=6
+                -DPHONON_BUILD_QT5=OFF
+                -DPHONON_BUILD_QT6=ON
+                -DPHONON_BUILD_SETTINGS=ON
+                -DKDE_INSTALL_USE_QT_SYS_PATHS=ON
+                -DKDE_INSTALL_DOCBUNDLEDIR="${EPREFIX}/usr/share/help"
+                -DPHONON_BUILD_DESIGNER_PLUGIN=$(usex designer)
+                -DCMAKE_DISABLE_FIND_PACKAGE_GLIB2=$(usex !pulseaudio)
+                -DCMAKE_DISABLE_FIND_PACKAGE_PulseAudio=$(usex !pulseaudio)
+              )
+
+              # Use make instead of ninja to avoid multiple outputs issue
+              export CMAKE_MAKEFILE_GENERATOR=emake
+              cmake_src_configure
+            }
+
+            src_install() {
+              cmake_src_install
+              make_desktop_entry "${PN}settings" \
+                "Phonon Audio and Video" preferences-desktop-sound
+            }
+
+phonon_vlc:
+  generator: builtin-dirlisting
+  template:
+    engine: helm
+
+  defaults:
+    files_dir: "{{ .Values.pn }}"
+    template: ../../templates/simple.tmpl
+    category: media-libs
+    dir:
+      url: 'https://download.kde.org/stable/phonon/phonon-backend-vlc/'
+      matcher: "[0-9.]+/$"
+      exclude: '[0-9]{5}'
+    assets:
+      - name: 'phonon-backend-vlc-{{ .Values.version }}.tar.xz'
+        url: 'https://download.kde.org/stable/phonon/phonon-backend-vlc/{{ .Values.version }}/phonon-backend-vlc-{{ .Values.version }}.tar.xz'
+    vars:
+      license: LGPL-2.1+ || ( LGPL-2.1 LGPL-3 )
+      slot: '0'
+      homepage: https://community.kde.org/Phonon
+      inherit:
+        - cmake
+        - flag-o-matic
+      iuse: debug
+      depend: |
+        dev-qt/qtbase:6[gui]
+        >=media-libs/phonon-4.12.0
+        media-video/vlc:=[dbus,ogg,vorbis(+)]
+      rdepend: '${DEPEND}'
+      bdepend: |
+        dev-libs/libpcre2:*
+        dev-qt/qttools:6[linguist]
+        >=kde-frameworks/extra-cmake-modules-5.115.0:*
+        virtual/pkgconfig
+
+  packages:
+    - phonon-vlc:
+        vars:
+          desc: VLC backend for the Phonon multimedia library
+          body: |
+            MY_PN="phonon-backend-vlc"
+            S="${WORKDIR}/${MY_PN}-${PV}"
```

Comment:
You can replace `MY_PN` and  `S`.
1) `S` could be defined as `s` vars eventually.
2) probably you can replace boh with this:
```yaml
vars:
   s: '${WORKDIR}/phonon-backend-vlc-{{ .Values.version }}'
```